[2022-08-06 19:38:15,005] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 19:38:15,031] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 19:38:15,031] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:15,031] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:15,031] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:15,054] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-02 00:00:00+00:00
[2022-08-06 19:38:15,062] {standard_task_runner.py:52} INFO - Started process 2152 to run task
[2022-08-06 19:38:15,066] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-02T00:00:00+00:00', '--job-id', '653', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvwmn77bv', '--error-file', '/tmp/tmpo6quabp8']
[2022-08-06 19:38:15,066] {standard_task_runner.py:80} INFO - Job 653: Subtask second_task
[2022-08-06 19:38:15,169] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:15,275] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T00:00:00+00:00
[2022-08-06 19:38:15,277] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:15,277] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:15,296] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:15,299] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:15,299] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:15,356] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220302T000000, start_date=20220806T193815, end_date=20220806T193815
[2022-08-06 19:38:15,400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:15,455] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:05,949] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 21:52:05,964] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 21:52:05,964] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:05,964] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:05,965] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:05,988] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-02 00:00:00+00:00
[2022-08-06 21:52:06,025] {standard_task_runner.py:52} INFO - Started process 2325 to run task
[2022-08-06 21:52:06,029] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-02T00:00:00+00:00', '--job-id', '651', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp1oi79p63', '--error-file', '/tmp/tmp48p7j766']
[2022-08-06 21:52:06,029] {standard_task_runner.py:80} INFO - Job 651: Subtask second_task
[2022-08-06 21:52:06,115] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:06,222] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T00:00:00+00:00
[2022-08-06 21:52:06,223] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:06,223] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:06,238] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:06,239] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:06,240] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:06,282] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220302T000000, start_date=20220806T215205, end_date=20220806T215206
[2022-08-06 21:52:06,321] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:06,362] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:29,673] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 21:57:29,689] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 21:57:29,689] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:29,689] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:29,689] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:29,709] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-02 00:00:00+00:00
[2022-08-06 21:57:29,716] {standard_task_runner.py:52} INFO - Started process 2116 to run task
[2022-08-06 21:57:29,719] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-02T00:00:00+00:00', '--job-id', '650', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvgaetcnh', '--error-file', '/tmp/tmp7rwhmycu']
[2022-08-06 21:57:29,719] {standard_task_runner.py:80} INFO - Job 650: Subtask second_task
[2022-08-06 21:57:29,800] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:29,881] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T00:00:00+00:00
[2022-08-06 21:57:29,881] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:29,882] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:29,893] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:29,895] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:29,895] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:29,929] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220302T000000, start_date=20220806T215729, end_date=20220806T215729
[2022-08-06 21:57:29,971] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:30,010] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:32,854] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 22:12:32,872] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 22:12:32,872] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:32,872] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:32,872] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:32,890] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-02 00:00:00+00:00
[2022-08-06 22:12:32,897] {standard_task_runner.py:52} INFO - Started process 2136 to run task
[2022-08-06 22:12:32,900] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-02T00:00:00+00:00', '--job-id', '656', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp4yh5jy3g', '--error-file', '/tmp/tmp91726lug']
[2022-08-06 22:12:32,900] {standard_task_runner.py:80} INFO - Job 656: Subtask second_task
[2022-08-06 22:12:33,002] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:33,108] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T00:00:00+00:00
[2022-08-06 22:12:33,109] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:33,110] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:33,128] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:33,129] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:33,130] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:33,165] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220302T000000, start_date=20220806T221232, end_date=20220806T221233
[2022-08-06 22:12:33,192] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:33,246] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:53,498] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 22:23:53,510] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [queued]>
[2022-08-06 22:23:53,510] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:53,510] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:53,510] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:53,526] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-02 00:00:00+00:00
[2022-08-06 22:23:53,531] {standard_task_runner.py:52} INFO - Started process 2184 to run task
[2022-08-06 22:23:53,533] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-02T00:00:00+00:00', '--job-id', '653', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmppb7ebxyd', '--error-file', '/tmp/tmpbwqg7sra']
[2022-08-06 22:23:53,534] {standard_task_runner.py:80} INFO - Job 653: Subtask second_task
[2022-08-06 22:23:53,589] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-02T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:53,668] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T00:00:00+00:00
[2022-08-06 22:23:53,669] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:53,669] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:53,680] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:53,682] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:53,682] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:53,706] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220302T000000, start_date=20220806T222353, end_date=20220806T222353
[2022-08-06 22:23:53,745] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:53,773] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
